pyphare/pyphare/pharein/maxwellian_fluid_model.py

```
@@ -301,3 +307,85 @@ def getCoord(L, R, idir):
+        for idir in np.arange(sim.ndim):
+            phare_utilities.debug_print("validating direction ...", idir)
+            if sim.boundary_types[idir] == "periodic":
+                phare_utilities.debug_print(f"direction {idir} is periodic?")
+                dual_left = (np.arange(-nbrDualGhosts, nbrDualGhosts) + 0.5) * sim.dl[0]
+                dual_right = dual_left + domain[0]
+                primal_left = np.arange(-nbrPrimalGhosts, nbrPrimalGhosts) * sim.dl[0]
+                primal_right = primal_left + domain[0]
+
+                direction = directions[idir]
+
+                for b_i, b_name in zip((bx, by, bz), ("Bx", "By", "Bz")):
```

Ruff (0.15.2)

[warning] Loop control variable `b_i` not used within loop body

(B007)